src/main/java/sirius/biz/tycho/kb/KnowledgeBaseController.java

```diff
@@ -36,6 +45,12 @@ public class KnowledgeBaseController extends BizController {
     @Part
     private EventRecorder eventRecorder;
 
+    @Part
+    private Resources resources;
+
+    @ConfigValue("http.response.defaultStaticAssetTTL")
+    private static Duration defaultStaticAssetTTL;
```

Comment:
Konfigwert fehlt in den .conf Dateien.

Reply:
Der kommt aus Web: https://github.com/scireum/sirius-web/blob/cb2399765c8274f79c3ba4a1c06e3801e7ced118/src/main/resources/component-065-web.conf#L282